canonical-web-and-design/webteam.canonical.com
# Add webteam practices as discourse docs

Add a practices section powered by discourse.

Fixes https://github.com/canonical-web-and-design/webteam.canonical.com/issues/91
Fixes #96 

# QA
- set the discourse credentials in `.env.local`
- dotrun
- got to `/practices`
- navigate a bit and check it works
- fiddle with the index topic to improve anything you want
